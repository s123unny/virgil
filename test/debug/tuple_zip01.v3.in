b ../core/tuple_zip01.v3 5
b ../core/tuple_zip01.v3 8
b ../core/tuple_zip01.v3 14
run
bt
info l
c
bt
info l
c
bt
info l
c
q
