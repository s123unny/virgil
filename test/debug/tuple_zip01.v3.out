breakpoint|../core/tuple_zip01.v3 5|0
breakpoint|../core/tuple_zip01.v3 8|1
breakpoint|../core/tuple_zip01.v3 14|2
stop|stopOnBreakpoint
bt|tuple_zip01.zip|../core/tuple_zip01.v3|8
bt|tuple_zip01.main|../core/tuple_zip01.v3|4
variable|0|this|tuple_zip01|tuple_zip01|false
variable|1|a|#1:Array<int>|Array<A>|true
variable|2|b|#2:Array<int>|Array<B>|true
stop|stopOnBreakpoint
bt|tuple_zip01.zip|../core/tuple_zip01.v3|14
bt|tuple_zip01.main|../core/tuple_zip01.v3|4
variable|0|this|tuple_zip01|tuple_zip01|false
variable|1|a|#1:Array<int>|Array<A>|true
variable|2|b|#2:Array<int>|Array<B>|true
variable|3|r|#3:Array<(int, int)>|Array<(A, B)>|true
variable|4|i|3|int|false
stop|stopOnBreakpoint
bt|tuple_zip01.main|../core/tuple_zip01.v3|5
variable|0|this|tuple_zip01|tuple_zip01|false
variable|1|arg|0|int|false
variable|2|r|#3:Array<(int, int)>|Array<(int, int)>|true
stop|end
result|3
